tools/c7n_org/c7n_org/cli.py

```diff
@@ -281,7 +281,8 @@ def filter_accounts(accounts_config, tags, accounts, not_accounts=None):
     accounts = comma_expand(accounts)
     not_accounts = comma_expand(not_accounts)
     for a in accounts_config.get('accounts', ()):
-        account_id = a.get('account_id') or a.get('project_id') or a.get('subscription_id') or ''
```

Comment:
why is this being removed?

Reply:
It's replaced with below `account_id = a.get('account_id') or ''`.

As commented here https://github.com/cloud-custodian/cloud-custodian/pull/9285#discussion_r1493900591, at this point, `project_id`, `subscription_id` or id of Oracle Cloud are no longer available after the normalization. That's also why the Oracle Cloud PR didn't change this line of code.